packages/blockchain-wallet-v4-frontend/src/data/components/swap/model.ts

```diff
@@ -5,3 +5,8 @@ export const getInput = (order: SwapOrderType) =>
 
 export const getOutput = (order: SwapOrderType) =>
   order.pair.split('-')[1] as CoinType
+
+export const getInputFromPair = (pair: string) => pair.split('-')[0] as CoinType
+
+export const getOutputFromPair = (pair: string) =>
+  pair.split('-')[1] as CoinType
```

Comment:
(minor) `getOutput` and  `getOutputFromPair` are using same approach we might call getOutputFromPair from getOutput like

```
export const getOutput = (order: SwapOrderType) => getOutputFromPair(order.pair)
```

and reuse code.